package.json

```diff
@@ -7,40 +7,45 @@
     "start": "yarn build && node build/src/index.js",
     "start:example": "node build/example/index.js",
     "clean": "rm -Rf build",
-    "build": "babel src --out-dir build/src --presets es2015,stage-2,react,babel-polyfill --plugins syntax-decorators",
-    "build:example": "yarn build && babel example --out-dir build/example --presets es2015,stage-2,react,babel-polyfill --plugins syntax-decorators",
-    "build:test": "yarn build && babel tests --out-dir build/tests --presets es2015,stage-2,react,babel-polyfill --plugins syntax-decorators  && cp -R tests/result build/tests/",
-    "build:watch": "babel src -w --out-dir build/src --presets es2015,stage-2,react,babel-polyfill --plugins syntax-decorators",
+    "build": "babel src --out-dir build/src",
+    "build:example": "yarn build && babel example --out-dir build/example",
+    "build:test": "yarn build && babel tests --out-dir build/tests && cp -R tests/result build/tests/",
+    "build:watch": "babel src -w --out-dir build/src",
     "prepublish": "yarn clean && yarn test && yarn clean && yarn build",
-    "test": "yarn build:test && node build/tests/index.js",
+    "lint": "eslint .",
+    "test": "yarn build:test && node build/tests/index.js && yarn lint",
```

Comment:
I didnt do that on purpose, because if tests fail I sometimes want to comment to bisect here an error is located. Having the linter in front will break that (as you will get a bunch of unused variables for example). Plus the test suite takes a few seconds to run, so I didnt found any linting errors after the test to be annoying

Reply:
To be clear, the test suite, including babel, takes 5 to 6 sec to run locally

Reply:
Fair enough :+1: